amgut/webserver.py

```diff
@@ -86,6 +88,10 @@ def __init__(self):
             (r"/authed/open-humans/", OpenHumansHandler),
             (r"/authed/connect/open-humans/", OpenHumansLoginHandler),
             (r"/authed/download/(.*)", DownloadHandler),
+            # (r"/authed/single/", SingleSampleHandler),
```

Comment:
Noted, thanks!

On (Apr-18-16|17:15), Joshua Shorenstein wrote:
> > @@ -86,6 +88,10 @@ def __init__(self):
> >              (r"/authed/open-humans/", OpenHumansHandler),
> >              (r"/authed/connect/open-humans/", OpenHumansLoginHandler),
> >              (r"/authed/download/(.*)", DownloadHandler),
> > +            # (r"/authed/single/", SingleSampleHandler),
> 
> It's a reminder that all the code exists for that to happen, once emperor stabilizes.
> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/american-gut-web/pull/583/files/664679af3caf1d792ffb6108e29a04582805d8a1#r60156303